lib/src/view/menu.dart

```diff
@@ -386,6 +386,20 @@ with some default direction chosen. Play with it and see!
 '''
         ..keyboard_shortcut = 'Ctrl+Shift+V')(),
       ///////////////////////////////////////////////////////////////
+      // select all
+      DropdownDivider({}),
+      (MenuDropdownItem()
+        ..on_click = ((_) => window.dispatchEvent(new KeyEvent('keydown', keyCode: KeyCode.A, ctrlKey: true).wrapped))
+        ..display = 'Select All'
+        ..tooltip = '''\
+Select all strands in the design.'''
+        ..keyboard_shortcut = 'Ctrl+A')(),
+      (MenuDropdownItem()
```

Comment:
Can we also make a keyboard shortcut Ctrl+Shift+A for this?

See view/design.dart around line 605 for an example of how to handle this:

```dart
// can paste even if nothing selected or not in select mode, if something is in copy buffer
if ((ev.ctrlKey || ev.metaKey) && key == KeyCode.V && !ev.shiftKey) {
  // Ctrl+V
  paste_strands_manually();
} else if ((ev.ctrlKey || ev.metaKey) && ev.shiftKey && key == KeyCode.V) {
  // Ctrl+Shift+V
  paste_strands_auto();
}
```